spec/models/manageiq/providers/hawkular/middleware_manager/middleware_server_spec.rb

```diff
@@ -0,0 +1,51 @@
+describe ManageIQ::Providers::Hawkular::MiddlewareManager::MiddlewareServer do
+  before do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
+    @ems_hawkular = FactoryGirl.create(:ems_hawkular,
```

Comment:
instead of `@ems_hawkular` you can use 
```ruby
let(:ems_hawkular) { FactoryGirl.create... }
...
ems_hawkular.reload
```
see [better_specs](http://betterspecs.org/#let)